### PR TITLE
r? ianbicking Fixes #566 Implement the full page button in css instead of with an image.

### LINKED ISSUE #566: "Full Page" asset
I tried to make the "Full Page" icon from the mockups, but I was having trouble making the background around the edges transparent. Can you create this asset for me.

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: server/src/views/frame.js [hunk 1]
```diff
@@ -241,8 +242,11 @@ class Frame extends React.Component {
           { nextClipNode }
         </div>
         <div id="full-page-button">
-          <img src={ this.props.staticLink("img/full-page-button.png") }
-            onClick={ this.clickFullPageButton.bind(this) } />
+          <a href="#" className="full-page-button-styles" onClick={ this.clickFullPageButton.bind(this) }>
```
Comment (ianb) [inline server/src/views/frame.js]: It occurs to me this could actually be `<a href="#frame">` or something similar to that, with no Javascript.

Comment (ianb) [inline server/src/views/frame.js]: Though it also occurs that exactly `#frame` wouldn't be right, since we want a buffer.

Comment (fzzzy) [inline server/src/views/frame.js]: Yeah, good point

Comment (fzzzy) [inline server/src/views/frame.js]: Is that really what we want though? We do some math to get it just how we want it

Comment (ianb) [inline server/src/views/frame.js]: Yes, that is what occurred to me after the first comment ;)

**Pull request merged**